### PR TITLE
Self-Service Listing Section Update

### PR DESCRIPTION
**Pull request by sofinico**

### CODE REVIEW

File: product-features/liquidity-pool/self-service-listing.md [hunk 1]
```diff
@@ -1,122 +1,101 @@
...
+Self-service listing allows you to list your own token on ALEX's decentralized exchange, enabling the permissionless trade of your token against a so-called anchor token within the exchange.
 
-Step 3: Deposit the listing token (Token X)
+ALEX decentralized exchange (DEX) is an Automated Market Maker (AMM) type of exchange. This means it operates with trading pools. These pools consist of a pair of tokens with sufficient funds (liquidity) of each to ensure a fair trade between the tokens that make up the pool.
 
-Step 4: Open Pool
+In this context, listing a token involves launching a new liquidity pool containing the new token and the anchor token on ALEX's AMM DEX, thereby facilitating trading within the community and enabling price discovery. 
 
-Step 5: Provide additional token information
+{% hint style="warning" %}
```
Comment (XianPaz) [inline product-features/liquidity-pool/self-service-listing.md:17]: se ve en el preview sin aplicar el estilo

File: product-features/liquidity-pool/self-service-listing.md [hunk 2]
```diff
@@ -1,122 +1,101 @@
...
+### What is self-service listing?
 
-Step 2: Waiting for the pool contract creation (around 24-48 hours)
+Self-service listing allows you to list your own token on ALEX's decentralized exchange, enabling the permissionless trade of your token against a so-called anchor token within the exchange.
 
-Step 3: Deposit the listing token (Token X)
+ALEX decentralized exchange (DEX) is an Automated Market Maker (AMM) type of exchange. This means it operates with trading pools. These pools consist of a pair of tokens with sufficient funds (liquidity) of each to ensure a fair trade between the tokens that make up the pool.
 
-Step 4: Open Pool
+In this context, listing a token involves launching a new liquidity pool containing the new token and the anchor token on ALEX's AMM DEX, thereby facilitating trading within the community and enabling price discovery.
```
Comment (XianPaz) [inline product-features/liquidity-pool/self-service-listing.md:15]: describe what is an anchor token.

File: product-features/liquidity-pool/self-service-listing.md (hunk 1 above)
Comment (sofinico) [reply product-features/liquidity-pool/self-service-listing.md:17]: Sip, porque es para Gitbook específico y Github no lo renderea. Lo vamos a comprobar cuando mergeen alexgo-io/alexlab-doc#1 🤞

**Comment by sofinico** [comment]
Pushié lo último que tengo. Falta que ALEX responda dudas que tengo en particular para la parte de los Steps y el wrapped contract.

Copio mensaje de tg:

> Steps (tentative):
>
> 
> 1. Submit the token information on self-listing page on ALEX Lab App & deposit the anchor token. 
> 
> 2. Wait for ALEX approval and wrapped token contract creation (around 24-48 hours).
> 
> 3. Deposit the listing token and the pool will be (automatically?) created on ALEX's AMM DEX. The new pool will be listed as an ALEX Pool in the Self Listed tab.
> 
> 4. Provide additional token information and to make the token visible on the ALEX Token List by clicking "Customer Support" on the self-listing page.
> 
> DOUBT A: are there two instances of deposit? first anchor token and once wrapped version is deployed, the user transfers the listed token in a different step? As the function request-create-and-fund suggests both fundings can be done in step 1, but it is a decision on how you want the user to interact.
> 
> DOUBT B: Once the wrapped contract is created and user has deposit both tokens, there is a waiting period until the pool is ready to use? how long is it?
> 
> DOUBT C: How does the procedure change when no wrapped version is needed? Since an approval on ALEX side is always necessary, step 2 will be just "Wait for token approval". Same doubt A applies here. Also, would you like to include the wrapped contract technicality or maybe describe just one procedure and doesn't mention the wrapped contract (leave step 2 just like waiting for token approval)?

File: product-features/liquidity-pool/self-service-listing.md [hunk 3]
```diff
@@ -1,122 +1,101 @@
...
-Step 5: Provide additional token information
+{% hint style="warning" %}
+**Supported tokens:** ALEX Self-Service Listing currently supports Stacks Chain Tokens (SIP-10 Standard Token).
+{% endhint %}
 
-***
+### Minimum requirements
 
-## Example:  Self-service listing for “STX-PEPE” pool
+👉 **Token Deployment.** Have your token deployed on Stacks blockchain since you will need you provide the token contract.
```
Comment (XianPaz) [inline product-features/liquidity-pool/self-service-listing.md]: typo: "you will need to provide the token contract"

File: product-features/liquidity-pool/self-service-listing.md (hunk 2 above)
Comment (sofinico) [reply product-features/liquidity-pool/self-service-listing.md:15]: @XianPaz fijate que ahí hice push addressando esto, decime qué te parece. Sobre todo si está bien dónde lo puse: https://github.com/CoinFabrik/alexlab-doc/pull/2/commits/de72019463af8fcbd93bddeff688c3f71c5893a4

File: product-features/liquidity-pool/self-service-listing.md (hunk 1 above)
Comment (sofinico) [reply product-features/liquidity-pool/self-service-listing.md:17]: Chequeado ✅ => https://docs.alexlab.co/product-features/liquidity-pool

**Pull request closed without merging**